src/ray/object_manager/plasma/create_request_queue.cc

```diff
@@ -168,6 +168,14 @@ Status CreateRequestQueue::ProcessRequests() {
     if (spilling_required) {
       spill_objects_callback_();
     }
+	//Block and evict tasks are called if the object store reaches over a threshold
+	if(RayConfig::instance().enable_BlockTasks() && block_tasks_required){
+        on_object_creation_blocked_callback_(lowest_pri, true, false);
```

Comment:
I think we may have talked about this earlier, but can we change this so that we only call the callback once? This is to reduce the chance of a race condition since these callbacks are async.